Cargo.lock

```diff
@@ -2472,9 +2472,9 @@ dependencies = [
 
 [[package]]
 name = "proc-macro2"
-version = "1.0.56"
+version = "1.0.64"
```

Comment:
Why does this change? 🤔

Reply:
The nightly CI build was broken without it.

Reply:
See https://github.com/rust-lang/rust/issues/113152

Reply:
I see. Thanks!